### PR TITLE
Replaced fwlinks in the docs/standard folder

### LINKED ISSUE #3425: Replace FwLinks with the actual target links - standard folder
sub-issue for folder https://github.com/dotnet/docs/tree/master/docs/standard

### PR DESCRIPTION
**Pull request by pkulikov**
Replaced remaining fwlinks in the **standard** folder

Fixes #3425

### CODE REVIEW

File: docs/standard/garbage-collection/performance.md [hunk 1]
```diff
@@ -45,8 +45,6 @@ ms.workload:
 ### Debugging with SOS  
  You can use the [Windows Debugger (WinDbg)](/windows-hardware/drivers/debugger/index) to inspect objects on the managed heap.  
   
- To install WinDbg, install Debugging Tools for Windows from the [WDK and Developer Tools Web site](http://go.microsoft.com/fwlink/?LinkID=103787).
```
Comment (pkulikov) [inline docs/standard/garbage-collection/performance.md]: This link was broken and the page linked in the sentence above contains installation notes.

Comment (mairaw) [reply docs/standard/garbage-collection/performance.md]: I think we could link to here: https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/debugger-download-tools

Comment (pkulikov) [reply docs/standard/garbage-collection/performance.md]: @mairaw that's true; fixed!

**Review by mairaw — APPROVED** [review]
Thank you @pkulikov! Changes look good. I contacted someone on my team to follow-up with the ECMA international site, since it seems to be down.

**Pull request merged**